adapter_docs/quickstart.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-Currently, *adapter-transformers* adds adapter components to all transformer models based on BERT and using PyTorch.
+Currently, *adapter-transformers* adds adapter components to the transformer models BERT, RoBERTa, XLM-RoBERTa, DistillBERT, GPT-2 and BART.
```

Comment:
Maybe we shouldn't keep the list of supported models here as this can get outdated quickly.

```suggestion
Currently, *adapter-transformers* adds adapter components to the PyTorch implementations of all transformer models listed in the *Supported Models* section.
```